packages/server/tests/acceptance/htsPrecompile/precompileCalls.spec.ts

```diff
@@ -603,6 +609,166 @@ describe('@precompile-calls Tests for eth_call with HTS', async function () {
     });
   });
 
+  describe('Create HTS token via direct call to Hedera Token service', async () => {
+    it('calls createFungibleToken', async () => {
+      const myImmutableFungibleToken = {
+        name: 'myImmutableFungibleToken',
+        symbol: 'MIFT',
+        treasury: accounts[0].wallet.address, // The key for this address must sign the transaction or be the caller
+        memo: 'This is an immutable fungible token created via the HTS system contract',
+        tokenSupplyType: true, // true for finite, false for infinite
+        maxSupply: 1000000,
+        freezeDefault: false, // true to freeze by default, false to not freeze by default
+        tokenKeys: [], // No keys. The token is immutable
+        expiry: {
+          second: 0,
+          autoRenewAccount: accounts[0].wallet.address,
+          autoRenewPeriod: 8000000,
+        },
+      };
+      const contract = new ethers.Contract(HTS_SYTEM_CONTRACT_ADDRESS, IHederaTokenServiceJson.abi, accounts[0].wallet);
+      const tx = await contract.createFungibleToken(myImmutableFungibleToken, 100, 18, {
+        value: BigInt(createTokenCost),
+        gasLimit: 10_000_000,
+      });
+      const receipt = await tx.wait();
+
+      expect(receipt.address).to.not.equal(HTS_SYTEM_CONTRACT_ADDRESS);
```

Comment:
Yes, we need to confirm for all the selectors that the proper token address is returned

Reply:
@quiet-node Do i understand correctly you suggest to extract the token address from the receipt and then assert to its the same as the receipt.address, which is essentially the same? I think this is kinda contradictory. 
It would make more sense to maybe take the token address from the mirror node and compare to the receipt.address, since this task is about exactly that - changing the receipt.address from the system contract to the actual token

Reply:
Yeah just an extra assertion that the receipt.address now is a tokenAddress. What you have now lgtm!